core/src/sidebar/sidebar.stories.tsx

```diff
@@ -33,77 +28,131 @@ const typeLozenge = (
-		{
-			title: "Item title",
-			path: "/1102",
-			after: typeBadge,
-		},
-		{
-			title: "Item title",
-			path: "/1103",
-			after: typeLozenge,
-		},
-	],
-};
+const bodyData: ISection[] = [
+	{
+		title: "subsection",
+		data: [
+			{
+				title: "Item title",
+				path: "/1000",
+				iconNormal: (
+					<Icon display="inline" component={outline.DownloadClound} />
+				),
+				iconActive: (
+					<Icon display="inline" component={solid.DownloadClound} />
+				),
+				childrens: [
+					{ title: "Item title", path: "/1101" },
+					{
+						title: "Item title",
+						path: "/1102",
+						after: typeBadge,
+					},
+					{
+						title: "Item title",
+						path: "/1103",
+						after: typeLozenge,
+					},
+				],
+			},
+			{
+				title: "Item title",
+				path: "/2000",
+				after: typeBadge,
+				iconNormal: (
+					<Icon display="inline" component={outline.DownloadClound} />
+				),
+				iconActive: (
+					<Icon display="inline" component={solid.DownloadClound} />
+				),
+				childrens: [
+					{ title: "Item title", path: "/2101" },
+					{
+						title: "Item title",
+						path: "/2102",
+						after: typeBadge,
+					},
+					{
+						title: "Item title",
+						path: "/2103",
+						after: typeLozenge,
+					},
+				],
+			},
 
-const menu2: IMenu = {
-	title: "Item title",
-	path: "/2000",
-	after: typeBadge,
-	iconNormal: <Icon display="inline" component={outline.DownloadClound} />,
-	iconActive: <Icon display="inline" component={solid.DownloadClound} />,
-	childrens: [
-		{ title: "Item title", path: "/2101" },
-		{
-			title: "Item title",
-			path: "/2102",
-			after: typeBadge,
-		},
-		{
-			title: "Item title",
-			path: "/2103",
-			after: typeLozenge,
-		},
-	],
-};
+			{
+				title: "Item title",
+				path: "/3000",
+				after: typeLozenge,
+				iconNormal: (
+					<Icon display="inline" component={outline.DownloadClound} />
+				),
+				iconActive: (
+					<Icon display="inline" component={solid.DownloadClound} />
+				),
+				childrens: [
+					{ title: "Item title", path: "/3101" },
+					{
+						title: "Item title",
+						path: "/3102",
+						after: typeBadge,
+					},
+					{
+						title: "Item title",
+						path: "/3103",
+						after: typeLozenge,
+					},
+				],
+			},
+		],
+	},
+];
 
-const menu3: IMenu = {
-	title: "Item title",
-	path: "/3000",
-	after: typeLozenge,
-	iconNormal: <Icon display="inline" component={outline.DownloadClound} />,
-	iconActive: <Icon display="inline" component={solid.DownloadClound} />,
-	childrens: [
-		{ title: "Item title", path: "/3101" },
-		{
-			title: "Item title",
-			path: "/3102",
-			after: typeBadge,
-		},
-		{
-			title: "Item title",
-			path: "/3103",
-			after: typeLozenge,
-		},
-	],
-};
+//footer
+const footerData: IMenu[] = [
+	{
+		title: "Cài đặt",
+		path: "/setting",
+		iconNormal: <Icon display="inline" component={outline.Tool} />,
+		iconActive: <Icon display="inline" component={solid.Tool} />,
+		isChild: false,
+	},
+	{
+		title: "Phân quyền",
```

Comment:
Why title is "Phân quyền" but path is `/user` 🤔

Reply:
because i didn't know what "phân quyền" really is so I named it base on the icon name